src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -18,6 +18,7 @@ export type Props = {
     onRedo: () => void;
     onReply: () => void;
     onUndo: () => void;
+    popupRef: React.Ref<PopupBase>;
```

Comment:
Do we need to pass this `popupRef` as a prop? Could we use `React.forwardRef` instead?

Reply:
Yeah we can use forwardRef.